res/values/strings.xml

```diff
@@ -157,6 +157,7 @@
     <string name="switch_camera">Switch Camera</string>
     <string name="toggle_fullscreen">Toggle Full Screen Mode</string>
     <string name="location">Location</string>
+    <string name="locations">Locations</string>
```

Comment:
I don't understand, why add this string that seems to be used nowhere?

Reply:
it is used in some ios experiments, where the word "Map" is not really fitting when showing a number or "None" beside (not yet there, but planned): 

<img width=302 src=https://github.com/deltachat/deltachat-android/assets/9800740/2cc7413b-dc6d-4eb7-999e-916adfcf4a23> <img width=320 src=https://github.com/deltachat/deltachat-android/assets/9800740/97063ea6-ccb5-4fa0-89e0-13bd411dd484>

also, as we're talking about "sending/deleting/all/whatnot locations" also elsewhere, this seems to be a more natural fit.